src/store/annotations/actions.ts

```diff
@@ -48,3 +48,4 @@ export const fetchAnnotationsAction = createAsyncThunk<APICollection<Annotation>
 
 export const removeAnnotationAction = createAction<string>('REMOVE_ANNOTATION');
 export const setActiveAnnotationIdAction = createAction<string | null>('SET_ACTIVE_ANNOTATION_ID');
+export const setAnnotationsInitialized = createAction<boolean | null>('SET_ANNOTATIONS_INITIALIZED');
```

Comment:
`setIsInitialized` and `SET_IS_INITIALIZED`. We should keep naming consistent where possible to help with code discoverability.